TombEngine/Game/control/box.cpp

```diff
@@ -654,23 +637,30 @@ void CreatureUnderwater(ItemInfo* item, int depth)
 
 void CreatureFloat(short itemNumber) 
 {
-	auto* item = &g_Level.Items[itemNumber];
```

Comment:
DO NOT remove asterisks after auto if it results in pointer type. It is a violation of coding conventions!